src/lib/zkapps_examples/zkapps_examples.ml

```diff
@@ -177,19 +177,22 @@ module Account_update_under_construction = struct
           ~f:(Fn.flip Zkapp_account.Actions.push_to_data_as_hash)
     end
 
+    (** I think this represent a zkapp update? why have that in zkapps examples though? Maybe it's just a builder for creating examples? *)
     type t =
-      { public_key : Public_key.Compressed.var
-      ; token_id : Token_id.Checked.t
-      ; caller : Token_id.Checked.t
-      ; account_condition : Account_condition.t
-      ; update : Update.t
+      { public_key : Public_key.Compressed.var  (** pubkey *)
```

Comment:
This comment is not a useful convent. Delete.

Reply:
oh these comments are just for me, didn't clean the PR yet : o

Reply:
(sry should have written a warning, though the WIP was enough)

Reply:
thought*